os/booting-on-vmware.md

```diff
@@ -68,7 +64,12 @@ Run VMware Workstation GUI:
 3. Name your VM, then click `Import`
 4. (Press `Retry` *if* VMware Workstation raises an "OVF specification" warning)
 5. Edit VM settings if necessary
-6. Start your CoreOS VM
+6. Create a cloud-config image (see below) containing at least at least one valid SSH key using the instructions at:
```

Comment:
Can we find a way to link some text instead of the raw link? Like:

...one valid SSH key using [a config-drive](https://github.com/coreos/coreos-cloudinit/blob/master/Documentation/config-drive.md).